components/Main/index.js

```diff
@@ -0,0 +1,100 @@
+import React, { useState } from 'react'
+import { View, Text, TextInput, TouchableOpacity, ScrollView } from 'react-native'
+import Item from 'components/Item'
+import './index.styl'
+import { observer } from 'startupjs'
+
+export default observer(function App () {
+
+  const [input, setInput] = useState('')
+  const[item, setItem] = useState([])
+
+  const AddTodo = () => {
+    if(input) {
+      const date = new Date()
+      setItem([...item, {
+        text : input,
+        key : Date.now(),
```

Comment:
@TimmyByDeOverly id is appropriate name. Please try to use some id generator.

Reply:
done